services/frontend/src/styles/_mixins.scss

```diff
@@ -29,6 +30,15 @@ $default-percentage: 5%;
     }
 }
 
+@mixin hover-box-shadow() {
+    transition: box-shadow 2 ease-out;
```

Comment:
This is an invalid rule because `2` doesn't have a unit. But I doubt you want the transition to last 2 seconds, so... change appropriately.

Reply:
Yep, changed it to 
```scss
transition: box-shadow 2 ease-out;
```